server/etcdserver/raft.go

```diff
@@ -155,9 +158,21 @@ func newRaftNode(cfg raftNodeConfig) *raftNode {
 func (r *raftNode) tick() {
 	r.tickMu.Lock()
 	r.Tick()
+	r.tickElapsed++
+	if r.tickElapsed == math.MaxUint64 {
```

Comment:
Don't worry about int overflow. It will take a long time for etcd to overflow, I think there are many other things that will overflow first in etcd :p

Reply:
Make sense, it will take 10^10 years with the default 100ms hearbeat timeout to overflow..